http_test.go

```diff
@@ -14,11 +17,35 @@ func TestHttp(t *testing.T) {
 	TestingT(t)
 }
 
-var _ = &httpSuite{}
+var _ = Suite(&httpSuite{})
 
 type httpSuite struct{}
 
-func (s httpSuite) TestServeHTTP(c *C) {
+type hasPrefix struct {
+	*CheckerInfo
+}
+
+var Contains Checker = &hasPrefix{
+	&CheckerInfo{Name: "Contains", Params: []string{"obtained", "expected"}},
+}
+
+func (checker *hasPrefix) Check(params []interface{}, names []string) (result bool, error string) {
+	defer func() {
+		if v := recover(); v != nil {
```

Comment:
🤔 

This is really strange, how could `strings.Contains(params[0].(string), params[1].(string))` panic uncontrollably?

Reply:
params may not be strings 🙂

Reply:
But you could avoid the panic with 

```go
obtained, ok1 := params[0].(string)
expected, ok2 := params[1].(string)
if !(ok1 && ok2) {
    return false, "invalid type blah blah lah"
}
```

See the implementation of `IsTrue` for detail.

Reply:
OK